common/client-libs/validator-client/src/validator_api/mod.rs

```diff
@@ -1,16 +1,24 @@
-// Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
+// Copyright 2022 - Nym Technologies SA <contact@nymtech.net>
```

Comment:
The correct copy right would be 2021-2022 I think, but if that's not how we did it in the rest of the codebase then I guess it doesn't matter that much. Or we could start now :?

Reply:
Good question. @mmsinclair or @futurechimp any opinions on that?

Reply:
As discussed on keybase, @mmsinclair aggress with @neacsu and I'm gonna update that header accordingly.